benchmarks/test_peer_selection.py

```diff
@@ -22,81 +22,78 @@
     absolute_import, unicode_literals, print_function, division
 )
 
-import mock
-import random
+from tornado import gen, ioloop
```

Comment:
The original benchmark doesn't work for the new case, since the original one just keep calling peer's choose method. In peer heap's branch , the choose method will simply fetch the top of the heap which doesn't make sense. It is not measuring the whole peer heap logic, it just measures the heap's peek call.